modules/kvantum/hm.nix

```
@@ -0,0 +1,70 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.kvantum = {
+    enable = config.lib.stylix.mkEnableTarget "Kvantum" pkgs.stdenv.hostPlatform.isLinux;
+    enableDefaultIcons = lib.mkOption {
+      description = "Default QT Icons";
+      type = lib.types.bool;
+      default = true;
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.kvantum.enable (let
+    cfg = config.stylix.targets.kvantum;
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      mkdir -p $out/share/Kvantum/Base16Kvantum
+      cat ${kvconfig} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.kvconfig
+      cat ${svg} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.svg
+    '';
+  in {
+    home.packages = with pkgs;
+      [
+        qt5ct
+        libsForQt5.qtstyleplugin-kvantum
+        qt6Packages.qtstyleplugin-kvantum
+        kvantumPackage
+      ]
+      ++ (lib.optionals cfg.enableDefaultIcons [
+        papirus-icon-theme
+      ]);
+
+    qt = {
+      enable = true;
+      platformTheme = "qtct";
+    };
+
+    xdg.configFile."Kvantum/kvantum.kvconfig".source = (pkgs.formats.ini {}).generate "kvantum.kvconfig" {
+      General.theme = "Base16Kvantum";
+    };
+
```

```suggestion
   xdg.configFile."Kvantum/Base16Kvantum".source = "${kvantumPackage}/share/Kvantum/Base16Kvantum";
```
This seems to be required for kvantum to find the theme.
should fix the issue nonetrix was having in https://github.com/danth/stylix/pull/142#issuecomment-1856755713